test/studies/filerator/walk-par.chpl

```diff
@@ -0,0 +1,13 @@
+use Filerator;
+
+config const startdir = ".";
+config const topdown = true;
+config const depth = max(int);
+config const dotfiles = false;
```

Comment:
doesn't have to be changed for this commit, but it seems like this should be `hidden` instead of `dotfiles` 

There's a couple other tests that also use `dotfiles` in studies/filerator/, but those can also be updated later